lib/pure/math.nim

```diff
@@ -159,16 +133,8 @@ when not defined(JS):
   proc exp*(x: float): float {.importc: "exp", header: "<math.h>".}
     ## Computes the exponential function of `x` (pow(E, x))
 
-  proc frexp*(x: float, exponent: var int): float {.
-    importc: "frexp", header: "<math.h>".}
-    ## Split a number into mantissa and exponent.
-    ## `frexp` calculates the mantissa m (a float greater than or equal to 0.5
-    ## and less than 1) and the integer value n such that `x` (the original
-    ## float value) equals m * 2**n. frexp stores n in `exponent` and returns
-    ## m.
-
-  proc round*(x: float): int {.importc: "lrint", header: "<math.h>".}
-    ## Converts a float to an int by rounding.
+  proc round0(x: float): float {.importc: "round", header: "<math.h>".}
```

Comment:
That was intentional - the round0 functions (C and JS) are implementations for the different targets.  round* (further down the file) is exported.  If round is only passed one argument it does the same as round0.  I'll add a comment to that effect.